model_compression_toolkit/target_platform_capabilities/targetplatform2framework/attach2pytorch.py

```diff
@@ -97,7 +99,9 @@ def __init__(self):
             OperatorSetNames.L2NORM: [LayerFilterParams(torch.nn.functional.normalize,
                                                         Eq('p', 2) | Eq('p', None))],
             OperatorSetNames.SSD_POST_PROCESS: [],  # no such operator in pytorch
-            OperatorSetNames.COMBINED_NON_MAX_SUPPRESSION: []  # no such operator in pytorch
+            OperatorSetNames.COMBINED_NON_MAX_SUPPRESSION: [multiclass_nms, multiclass_nms_with_indices, MulticlassNMS,
```

Comment:
Why do we need multiclass_nms and multiclass_nms_with_indices if it is not support by fx trace ?